tools/ClassGen/InstrGen.cpp

```diff
@@ -440,6 +440,17 @@ int main(int argc, char **argv) {
       .autoVerify(VerifyKind::SameType, {"Dest", "Src"})
       .autoIRGen("Log");
 
+  BB.newInstr("ElementExp")
+      .addOperand("Dest", OperandKind::Out)
+      .addOperand("Src", OperandKind::In)
+      .inplaceOperand({
+          "Dest",
+          "Src",
+      })
+      .dataParallel()
+      .autoVerify(VerifyKind::SameType, {"Dest", "Src"})
```

Comment:
a bit confusing, why do you have auto verify and also custom verification logic for this node.

Reply:
This is for the Instr, the other verification is for the Node.

Reply:
right, missed that